api/views.py

```
@@ -0,0 +1,1021 @@
+        denial_response_info = (
+            common_view_logic.DenialCreatorHelper.create_or_update_denial(
+                denial=denial,
+                creating_professional=creating_professional,
+                **serializer_data,
+            )
+        )
+        denial = Denial.objects.get(uuid=denial_response_info.uuid)
+        # Creating a pending appeal
+        try:
+            Appeal.objects.get(for_denial=denial)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)